drivers/azure/azure.go

```
@@ -312,6 +313,8 @@ func (d *Driver) Start() error {
 		return err
 	}
 
+	d.IPAddress, _ = d.GetIP()
```

Why swallow the error?